docker/daphne/Dockerfile

```diff
@@ -1,36 +1,43 @@
 FROM ubuntu:16.04
 
 # Generate some locales
-RUN apt-get update && apt-get install -y locales && rm -rf /var/lib/apt/lists/* \
-    && localedef -i en_US -c -f UTF-8 -A /usr/share/locale/locale.alias en_US.UTF-8
+RUN set -eu && \
+    apt-get update && \
+    apt-get install -y locales && \
+    rm -rf /var/lib/apt/lists/* && \
+    localedef -i en_US -c -f UTF-8 -A /usr/share/locale/locale.alias en_US.UTF-8
+
 ENV LANG en_US.utf8
 
-# Install OS packages
-RUN apt-get update && apt-get install -y \
-    # Direct dependencies for socialhome
-    git python-virtualenv python3-setuptools python3-dev build-essential libpq-dev \
-    # Federation
-    libxml2-dev libxslt-dev lib32z1-dev \
-    # Redis TODO move to own container
-    redis-server \
-    # Generic stuff for Ubuntu
-    wget apt-transport-https apt-utils
+# Install OS packages:
+# - direct dependencies for Socialhome
```

Comment:
Having comments this way kinda makes them not so useful since the relation of the comment and what is being commented on is being lost. I would prefer keeping the comments before the line itself unless there is a technical reason that it doesn't work. The comment could postfix the line too if before doesn't work in some situation?

Reply:
When I build image with these inline comments, I get notification from docker (possibly newer version than yours) that this is deprecated and will stop working in future versions. This is the only reason I've moved them up.